junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedTestNameFormatter.java

```diff
@@ -45,7 +46,13 @@ String format(int invocationIndex, Object... arguments) {
 			arguments[i] = StringUtils.nullSafeToString(arguments[i]);
 		}
 
-		return MessageFormat.format(result, arguments);
+		try {
+			return MessageFormat.format(result, arguments);
+		}
+		catch (IllegalArgumentException ex) {
+			String message = "The naming pattern defined for the parameterized tests is invalid: " + ex.getMessage();
```

Comment:
Don't you think Java developers should be trained to do so anyway? Nevertheless, I would be ok with adding that sentence.